bin/commands/runs.js

```diff
@@ -9,14 +9,21 @@ const archiver = require("../helpers/archiver"),
   utils = require("../helpers/utils"),
   fileHelpers = require("../helpers/fileHelpers"),
   syncRunner = require("../helpers/syncRunner"),
-  reportGenerator = require('../helpers/reporterHTML').reportGenerator;
+  reportGenerator = require('../helpers/reporterHTML').reportGenerator,
+  {initTimeComponents, markBlockStart, markBlockEnd, getTimeComponents} = require('../helpers/timeComponents');
 
 module.exports = function run(args) {
   let bsConfigPath = utils.getConfigPath(args.cf);
   //Delete build_results.txt from log folder if already present.
+  initTimeComponents();
```

Comment:
Not needed according to current implementation

Reply:
this is to explicitly state that we're recording from here

Reply:
Then can we remove the extra initialization?

Reply:
The global initialization is put in place so that the function calls never break the flow and cause runtime error since this is purely instrumentation code. `initTimeComponents` is added here to start measuring time from this point.